src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -293,12 +290,12 @@
     <value>Can not access a closed Stream.</value>
   </data>
   <data name="net_http_libcurl_callback_notsupported" xml:space="preserve">
-    <value>The handler does not support custom handling of certificates with this combination of libcurl ({0}) and its SSL backend ("{1}").</value>
+    <value>The handler does not support custom validation of certificates with this combination of libcurl ({0}) and its SSL backend ("{1}"). A libcurl built with OpenSSL is required.</value>
```

Comment:
Right now on macOS using libcurl+openssl gains nothing.  And even if I can get my gedankenexperiment working today the clientcerts message will still be misleading (since it will make callback work, but not clientcerts).

So mainly I'm concerned with macOS users saying "it didn't work, and I followed the exception's instructions, and it gave me the exact same error".